lib/components/text_component.dart

```diff
@@ -38,19 +36,11 @@ class TextComponent extends PositionComponent {
     _tp = config.toTextPainter(_text);
     width = _tp.width;
     height = _tp.height;
-
-    _updateOffset(Position.empty());
-  }
-
-  void _updateOffset(Position p) {
```

Comment:
so basically you are saying this is doing twice the translation because PositionComponent is already translating when we set width/height rignt? let me just test this on the example app

Reply:
gotcha, that makes sense!